tileview/src/main/java/com/qozix/tileview/widgets/ZoomPanLayout.java

```diff
@@ -196,6 +197,15 @@ public void setSize( int width, int height ) {
     requestLayout();
   }
 
+  /**
+   * Sets the size of an extended border around the view.
+   *
+   * @param border  Extra size of the scaled view.
+   */
+  public void setExtraBorder(int border) {
+    mExtraBorder = border;
```

Comment:
this isn't really a "border", it's more like padding.  maybe call it something like "inset" or "visual padding"?

Reply:
Agreed. Naming has never been my strong suite ;)
I'll call it `setImagePadding` instead.